### PR TITLE
Add custom  commands accessible from makejdk-any-platform

### PR DESCRIPTION
**Pull request by d3r3kk**
Add the command line --make-args, similar to --config-args but for `make`

Fix for #1283

### CODE REVIEW

File: makejdk-any-platform.1 [hunk 1]
```diff
@@ -119,6 +119,9 @@ the one you are trying to build.
 .BR \-k ", " \-\-keep
 if using docker, keep the container after the build.
 .TP
+.BR \-C ", " \-\-make-args " " \fI<args>\fR
```
Comment (karianna) [inline makejdk-any-platform.1]: I think there's a confusion clash here as `-C` is used for `--configure-args` in _config_init.sh_?

File: sbin/common/config_init.sh [hunk 2]
```diff
@@ -406,8 +410,9 @@ function configDefaults() {
   # JVM variant, e.g. client or server, defaults to server
   BUILD_CONFIG[JVM_VARIANT]=${BUILD_CONFIG[JVM_VARIANT]:-""}
 
-  # Any extra args provided by the user
+    # Any extra config/make args provided by the user
```
Comment (karianna) [inline sbin/common/config_init.sh]: Whitespace

File: makejdk-any-platform.1 (hunk 1 above)
Comment (d3r3kk) [reply makejdk-any-platform.1]: copypasta - fixing

File: sbin/common/config_init.sh (hunk 2 above)
Comment (d3r3kk) [reply sbin/common/config_init.sh]: :+1:

File: sbin/common/config_init.sh [hunk 3]
```diff
@@ -406,8 +410,9 @@ function configDefaults() {
   # JVM variant, e.g. client or server, defaults to server
   BUILD_CONFIG[JVM_VARIANT]=${BUILD_CONFIG[JVM_VARIANT]:-""}
 
-  # Any extra args provided by the user
+    # Any extra config / make args provided by the user
```
Comment (karianna) [inline sbin/common/config_init.sh]: @d3r3kk - I think the whitespace indent is incorrect here.

**Pull request merged**